File: mypostbank-ui/src/app/services/models/card-dto.ts
/* tslint:disable */
/* eslint-disable */
export interface CardDto {
  accountId?: number;
  cardId?: number;
  cardNo?: string;
  createDate?: string;
  expirationDate?: string;
  owner?: string;
  status?: 'PENDING' | 'FROZEN' | 'ACTIVE' | 'BLOCKED';
  userId?: number;
}

File: src/main/java/com/Spring/MyPostBank/DTOs/CardDTO.java
package com.Spring.MyPostBank.DTOs;

import com.Spring.MyPostBank.Enums.CardStatus;
import lombok.AllArgsConstructor;
import lombok.Builder;
import lombok.Data;
import lombok.NoArgsConstructor;

import java.time.LocalDate;
import java.time.LocalDateTime;

@Data
@Builder
@AllArgsConstructor
@NoArgsConstructor
public class CardDTO {
    private Integer cardId;
    private String cardNo;
    private CardStatus status;
    private LocalDate expirationDate;
    private Integer accountId;
    private Integer userId;
    private LocalDateTime createDate;
    private String Owner;
}
